src/cljsjs/create_react_class.cljs

```diff
@@ -1,8 +0,0 @@
-(ns cljsjs.create-react-class
-  (:require
-    ["create-react-class" :as create-react-class]
-    ["react" :as react]))
-
-
-(js/goog.object.set react "createClass" create-react-class)
-(js/goog.exportSymbol "createReactClass" create-react-class)
```

Comment:
Why aren't cljsjs namespaces necessary anymore?

Reply:
They never were really necessary with `shadow-cljs`. But devcards was a bit broken and required them as a workaround for https://github.com/bhauman/devcards/issues/165.